src/components/body/content/Main.tsx

```diff
@@ -3,19 +3,19 @@ import Lessoner from "./lessoner/Lessoner";
 import Categories from "./categories/Categories";
 import Lessons from "./lessons/Lessons";
 import About from "./about/About";
-import SignIn from "./SignIn/SignIn";
-import SignUp from "./lessoner/SignUp/SignUp";
+import LoginPage from "../../../pages/LoginPage";
+import FirstRegistrationForm from "../../FirstRegistrationForm";
 
 const Content = () => {
- return (
+  return (
     <div className="main">
       <Routes>
+        <Route path="/users/sign_in" element={<LoginPage />} />
+        <Route path="/users/sign_up" element={<FirstRegistrationForm />} />
         <Route path="/" element={<Lessoner />} />
         <Route path="/categories" element={<Categories />} />
         <Route path="/lessons" element={<Lessons />} />
         <Route path="/about" element={<About />} />
-        <Route path="/users/sign_in" element={<SignIn />} />
-        <Route path="/users/sign_up" element={<SignUp />} />
       </Routes>
     </div>
   );
```

Comment:
please, not include that file into the PR

Reply:
remove this files